### DISCUSSION TITLE
Different execution modes possible? (Single / batch / parallel / random / scheduled)

### DISCUSSION THREAD

**Discussion by techtreet** [vmware/mangle#55] [Q&A]
What are all the different execution modes possible? (Single / batch / parallel / random / scheduled)

Single and Scheduled is obviously possible

Batch - Possible to group multiple fault scenarios and run them together so that they run one after the other?

Parallel - Possible to group multiple scenarios and run them all at once in parallel?

Random - Possible to group multiple scenarios and run them all one at a time randomly?

**Comment by ranjansvmware** [comment]
You can execute multiple different faults in parallel. Rest API call is supported by Mangle you can always use that to achieve your required scenarios. Also, you should be able to achieve Single, Batch and Parallel fault runs by scheduling them.
